vocabularies/trs.ttl

```diff
@@ -0,0 +1,178 @@
+# baseURI: http://linked.data.gov.au/def/trs
+# imports: http://www.w3.org/2004/02/skos/core
+# imports: http://www.w3.org/2006/time
+
+@prefix trs: <http://linked.data.gov.au/def/trs#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix time: <http://www.w3.org/2006/time#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/trs>
```

Comment:
Should "trs" be spelled out? **temporal-reference-system**

Reply:
I don't think so: no need within the identifier (the URI) as it's everywhere in the metadata.